test-cases/longevity/longevity-100gb-4h.yaml

```diff
@@ -17,3 +17,6 @@ nemesis_interval: 5
 
 user_prefix: 'longevity-100gb-4h'
 space_node_threshold: 64424
+
+send_email: true
+email_recipients: ['qa@scylladb.com']
```

Comment:
Why adding this to all yamls and not adding to test_defaults?

Reply:
@roydahan For that moment we have template for report only for gemini and longevity.
we will add templates to other test types, then this config will be moved to defaults

Reply:
we have also for performance.
So, I'm not sure which ones doesn't have the report.
It's ok to put it in the defaults and set "false" on those we don't have.

Reply:
@roydahan adding to defaults config
send_email: false
email_recipients: ['qa@scylladb.com']